DragaliaAPI.Database/Entities/DbPlayerDragonGift.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using DragaliaAPI.Shared.Definitions.Enums;
+using Microsoft.EntityFrameworkCore;
+
+namespace DragaliaAPI.Database.Entities;
+
+[Table("PlayerDragonGift")]
+[Index(nameof(DeviceAccountId))]
+public class DbPlayerDragonGift : IDbHasAccountId
+{
+    [Column("DeviceAccountId")]
+    [Required]
```

Comment:
fyi this doesn't do anything to the database, the column is always NOT NULL unless the type is nullable, i think this just causes a runtime exception if it is not specified